habitat_sim/agent/agent.py

```diff
@@ -166,31 +162,17 @@ def reconfigure(
         self.agent_config = agent_config
 
         if reconfigure_sensors:
-            self._sensors.clear()
```

Comment:
See my previous comments.
If you keep the `_sensors` concept, you do not have to change the code here.